.github/workflows/build-dev.yml

```
@@ -108,4 +108,4 @@ jobs:
           push: true
           tags: | 
             ${{ env.IMAGE_NAME }}:${{ env.VERSION }}
-            ${{ env.IMAGE_NAME }}:dev
+            ${{ env.IMAGE_NAME }}:dev
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)